diff --git a/rules.mk b/rules.mk
index f04f3f4..ede9f60 100644
--- a/rules.mk
+++ b/rules.mk
@@ -18,8 +18,8 @@ $(BUILD)/$(BIN).html: $(OBJS)
 	@echo HTML $@
 	@$(CC) $(LDFLAGS) $(OBJS) $(LIBS) -o $@ \
 		-s ASYNCIFY=1 \
-		-s EXPORTED_RUNTIME_METHODS=lengthBytesUTF8,printErr \
-		-s EXPORTED_FUNCTIONS=_main \
+		-s EXPORTED_RUNTIME_METHODS=[lengthBytesUTF8,printErr] \
+		-s EXPORTED_FUNCTIONS=[_main] \
 		--shell-file=$(TOP)/watch-library/simulator/shell.html
 
 $(BUILD)/$(BIN).elf: $(OBJS)
